yt/units/yt_array.py

```diff
@@ -589,39 +589,78 @@ def convert_to_mks(self):
         """
         return self.convert_to_units(self.units.get_mks_equivalent())
 
-    def in_units(self, units):
+    def in_units(self, units, equivalence=None, **kwargs):
```

Comment:
What does kwargs do here? If it does something useful then it needs to be documented in the docstring.

Reply:
If you look in the narrative part of the docstring there is a comment there about this.

Reply:
Can you break that sentence off into it's own paragraph so it's more obvious? Maybe emphasize it with an rst note admonition?